addons/website/views/snippets/snippets.xml

```diff
@@ -1004,24 +1004,27 @@
-                    <we-button data-select-label="Cards" title="Cards" data-select-template="website.s_mega_menu_cards" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_cards.svg">Cards</we-button>
-            </we-select>
+        <we-select string="Template" data-name="mega_menu_template_opt">
+            <we-button data-select-label="Multi Menus" title="Multi Menus" data-select-template="website.s_mega_menu_multi_menus" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_multi_menus.svg">Multi Menus</we-button>
+            <we-button data-select-label="Image Menu" title="Image Menu" data-select-template="website.s_mega_menu_menu_image_menu" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_menu_image_menu.svg">Image Menu</we-button>
+            <we-button data-select-label="Odoo Menu" title="Odoo Menu" data-select-template="website.s_mega_menu_odoo_menu" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_odoo_menu.svg">Odoo Menu</we-button>
+            <we-button data-select-label="Little Icons" title="Little Icons" data-select-template="website.s_mega_menu_little_icons" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_little_icons.svg">Little Icons</we-button>
+            <we-button data-select-label="Big Icons Subtitles" title="Big Icons Subtitles" data-select-template="website.s_mega_menu_big_icons_subtitles" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_big_icons_subtitles.svg">Big Icons Subtitles</we-button>
+            <we-button data-select-label="Images Subtitles" title="Images Subtitles" data-select-template="website.s_mega_menu_images_subtitles" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_images_subtitles.svg">Images Subtitles</we-button>
+            <we-button data-select-label="Logos" title="Logos" data-select-template="website.s_mega_menu_menus_logos" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_menus_logos.svg">Logos</we-button>
+            <we-button data-select-label="Thumbnails" title="Thumbnails" data-select-template="website.s_mega_menu_thumbnails" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_thumbnails.svg">Thumbnails</we-button>
+            <we-button data-select-label="Cards" title="Cards" data-select-template="website.s_mega_menu_cards" data-img="/website/static/src/img/snippets_thumbs/s_mega_menu_cards.svg">Cards</we-button>
+        </we-select>
         <we-select string="Size">
             <we-button data-select-class="">Full-Width</we-button>
             <we-button data-select-class="o_mega_menu_container_size">Narrow</we-button>
         </we-select>
     </div>
 
+    <div data-js="MegaMenuResetOnDelete" data-selector=".o_mega_menu > section > * " />
```

Comment:
@qsm-odoo We have this empty customize-block option now. What should I do about it? Hide it (I don't see straightforward way to do that)? Or let it as it is for potential future options.
![Screenshot from 2021-09-15 14-09-42](https://user-images.githubusercontent.com/49784121/133430816-b24bef4c-1014-4f0d-a40a-73f07b36edc3.png)

Reply:
Not sure where it comes from but yes, find a way to hide it of course (and do that in a dedicated commit if it is relevant)

Reply:
It is because the selector doesn't target the row anymore but the container just above. Should customize-block option without any visible option inside always be hidden? I imagine this is not the only case where we have such empty customize-block. So may be it is worth doing something generic for this case.
![Screenshot from 2021-09-15 14-35-04](https://user-images.githubusercontent.com/49784121/133434272-3c436be8-eb74-4b06-a82b-2bb15a757fc7.png)

Reply:
Why is there an option targeting the container, this is what I don't understand

Reply:
If the user removes manually all the cols inside it then the container will be removed and the onRemove called. We can't target the cols and reset the template if no neighbors due to this kind of template. The data-selector here is generic enough for all the mega_menu templates...
![Screenshot from 2021-09-15 14-54-44](https://user-images.githubusercontent.com/49784121/133437201-f5013c46-3893-4b3d-8aee-3bd53a21f00c.png)

Reply:
Why not targeting the section itself?

Reply:
oops indeed